styleguide/components/41-organisms/programme/programme.twig

```diff
@@ -0,0 +1,95 @@
+<div class="programme-wrapper">
+  <h2 class="visually-hidden">Programme</h2>
+  <dl class="programme box-top">
+    <div class="inner-box">
+
+      {% for slotKey, slot in slots %}
+        <div class="programme-slot" role="listitem">
+          <dt class="programme-slot-title">{{ slot.title }}</dt>
+          <dd>
+            <ul class="grid-3">
+
+            {% for itemKey, item in slot.items %}
+              {% set key = slotKey ~ itemKey %}
+              <li>
+                <!-- Programme detail -->
+                <section id="{{ key }}" class="programme-detail modal" tabindex="-1">
+                  <div class="programme-detail-inner">
+                    <div class="modal-actions">
+                      {% include '@modal-close' %}
+                    </div>
+
+                    <h2>{{ item.title }}</h2>
+                    {% if item.subtitle %}
+                      <h3>{{ item.subtitle }}</h3>
+                    {% endif %}
+
+                    {% if item.images|length > 1 %}
+                      {% include '@image-gallery--multiple' with {
+                        type: 'secondary',
+                        items: item.images
+                      } %}
+                    {% elseif item.images|length > 0  %}
+                      {% include '@image-gallery' with {
+                        type: 'secondary',
+                        src: item.images|first.src,
+                        alt: item.images|first.alt,
+                        caption: item.images|first.caption
+                      } %}
+                    {% endif %}
+
+                    {% if item.description %}
+                      {% include '@paragraph' with {
+                        'text': item.description
+                      } %}
+                   {% endif %}
+
+                    {% if item.video %}
+                      {% include '@video' with item.video %}
+                    {% endif %}
+                  </div>
+                </section>
+                <div class="programme-detail-overlay"></div>
```

Comment:
I believe you are missing the "modal__close" classe, because the modal doesn't close.